selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindTextFeatureTest.java

```diff
@@ -29,79 +28,127 @@
 import org.eclipse.che.selenium.pageobject.ProjectExplorer;
 import org.eclipse.che.selenium.pageobject.machineperspective.MachineTerminal;
 import org.openqa.selenium.Keys;
+import org.testng.Assert;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /** @author Aleksandr Shmaraev */
 public class FindTextFeatureTest {
 
-  private static final String PROJECT_NAME = NameGenerator.generate("FindTextFeature", 4);
+  private static final String PROJECT_NAME = "FindTextFeature";
 
-  private static final String FIND_NOTHING = "(0 occurrence)";
+  private static final String FIND_NOTHING = "Found occurrences of 'dddhhh' (0 occurrence)";
 
   private static final String PR_1_EXPECTED_TEXT_1 =
-      "(4 occurrences)\n"
-          + "AppController.java\n"
-          + "(/"
-          + PROJECT_NAME
-          + "/my-webapp/src/main/java/org/eclipse/qa/examples/AppController.java)\n"
-          + "SayHello.java\n"
-          + "(/"
-          + PROJECT_NAME
-          + "/my-lib/src/main/java/hello/SayHello.java)\n"
-          + "SayHelloTest.java\n"
-          + "(/"
-          + PROJECT_NAME
-          + "/my-lib/src/test/java/hello/SayHelloTest.java)\n"
-          + "guess_num.jsp\n"
-          + "(/"
-          + PROJECT_NAME
-          + "/my-webapp/src/main/webapp/WEB-INF/jsp/guess_num.jsp)";
+      "Found occurrences of 'Filesystem' (1 occurrence)\n"
+          + "/FindTextFeature/readme.con\n"
+          + "(1 occurrence of 'Filesystem' found)\n"
+          + "1:   Filesystem 1K-blocks Used Available Use% Mounted on";
+  private static final String PR_1_EXPECTED_TEXT_2 =
+      "Found occurrences of 'Feature' (1 occurrence)\n"
+          + "/FindTextFeature/readme.api\n"
+          + "(1 occurrence of 'Feature' found)\n"
+          + "1:   FindTextFeatureTest";
 
-  private static final String PR_2_EXPECTED_TEXT_1 = "(4 occurrences)";
+  private static final String PR_2_EXPECTED_TEXT_1 =
+      "Found occurrences of 'String' (9 occurrences)\n"
```

Comment:
We have **PROJECT_NAME** constant and such changes with real project name hardcoded in string make change of **PROJECT_NAME** quite hard. I suggest to use String.format() method here with substitution **PROJECT_NAME**.